packages/dds/tree/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -115,13 +115,15 @@ function invertMark(
 					id: mark.id,
 					cellId: outputId,
 					count: mark.count,
+					revision: mark.revision,
```

Comment:
Instead of using the original revision, `invertMark` should take a new revision parameter and use that. Otherwise this means that any changeset and its inverse will use the same RevisionTag.